tool/planet/etcd.go

```diff
@@ -700,6 +702,10 @@ func getEtcdDataDir() (string, error) {
 	return getBaseEtcdDir(version), nil
 }
 
+func getBaseEtcdDir(version string) string {
```

Comment:
Another question for @knisbet. With the previous implementation, it is puzzling when is the `/ext/etcd` return is useful (e.g. when `version == AssumeEtcdVersion`) unless I'm missing anything:
```go
func getBaseEtcdDir(version string) string {
	p := DefaultEtcdStoreBase
	if version != AssumeEtcdVersion {
		p = filepath.Join(DefaultEtcdStoreBase, version)
	}
	return p
}
```